File: C#/BluffinMuffin.Protocol/Game/PlayerSitInResponse.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// The response from the server on the SitInRequest
    /// </summary>
    public class PlayerSitInResponse : AbstractBluffinReponse<PlayerSitInCommand>, IGameCommand
    {
        /// <summary>
        /// The Seat number where the player will sit. It could be different from the one asked if the seat was not available. Wil have -1 if the player is not sitting anywhere.
        /// </summary>
        [ExampleValue(7)]
        public int NoSeat { get; set; }

        /// <summary>
        /// The id of the table where this command needs to go
        /// </summary>
        public int TableId
        {
            get { return Command.TableId; }
        }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public PlayerSitInResponse(PlayerSitInCommand command)
            : base(command)
        {
        }
    }
}

File: Python/BluffinMuffin.Protocol/game/player_sit_in_response.py
from abstract_game_command import AbstractGameResponse
from game.player_sit_in_command import PlayerSitInCommand

__author__ = 'ericmas001@gmail.com'



class PlayerSitInResponse(AbstractGameResponse):
    def __init__(self, obj):
        super().__init__(obj, PlayerSitInCommand(obj['Command']))
        self.no_seat = obj['NoSeat']

    def __str__( self ):
        return '{0} => {1}'.format(super().__str__(), self.no_seat)